src/main/java/lotto/domain/LottoNumber.java

```diff
@@ -0,0 +1,56 @@
+package lotto.domain;
+
+import lombok.Getter;
+
+import java.util.Objects;
+
+import static lotto.exception.ExceptionMessage.NON_INTEGER_INPUT_FOR_LOTTO_NUMBER;
+import static lotto.exception.ExceptionMessage.OUT_OF_BOUND_INPUT_FOR_LOTTO_NUMBER;
+import static lotto.util.NumberValidateUtils.isInteger;
+
+public class LottoNumber {
+
+    public static final int LOWER_BOUND = 1;
+    public static final int UPPER_BOUND = 45;
+
+    @Getter
+    private final int lottoNumber;
+
+    public LottoNumber(String input) throws IllegalArgumentException {
+        validate(input);
+
+        this.lottoNumber = Integer.parseInt(input);
+    }
+
+    private void validate(String input) throws IllegalArgumentException {
+        if (!isInteger(input)) {
```

Comment:
`isInteger` 는 직독하면 정수인가? 는 판단문입니다.
다만 내용을 보니 0~9까지 숫자 즉 음이 아닌 정수인지를 정규표현식을 이용해 판단하고 있네요.
isNonNegativeInteger 정도로 더 정확하게 표현해보면 어떨까요?
```suggestion
        if (!isNonNegativeInteger(input)) {
```

Reply:
넵! 더 명확하게 표현하도록 수정했습니다.